test/unit/schema-builder/cockroachdb.js

```diff
@@ -20,4 +20,56 @@ describe('CockroachDB SchemaBuilder', function () {
       'create table "uuid_primary" ("id" uuid primary key default gen_random_uuid())'
     );
   });
+
+  it('drop unique if exists', function () {
+    tableSql = client
+      .schemaBuilder()
+      .table('users', function (table) {
+        table.dropUniqueIfExists('foo');
+      })
+      .toSQL();
+    equal(1, tableSql.length);
+    expect(tableSql[0].sql).to.equal(
+      'drop index if exists "users"@"users_foo_unique" cascade '
```

Comment:
What's with the stray space at the end of this expectation? Is that an artifact of query construction or a typo? If it's a typo, why didn't the test fail?